components/python/pycurl/Makefile

```diff
@@ -27,17 +27,17 @@
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		pycurl
-COMPONENT_VERSION=	7.43.0.2
-COMPONENT_REVISION=	2
+COMPONENT_VERSION=	7.44.1
+COMPONENT_REVISION=	1
 COMPONENT_PROJECT_URL=	http://pycurl.sourceforge.net/
```

Comment:
COMPONENT_REVISION can be removed as COMPONENT_VERSION has changed.
As long as we don't release a new package version the COMPONENT_REVISION only needs to be updated once.